backend/src/schema/resolvers/reports.spec.js

```diff
@@ -1,37 +1,73 @@
 import { GraphQLClient } from 'graphql-request'
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-14T09:40:09Z" title="Monday, October 14th 2019, 11:40:09 am +02:00">Oct 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,37 +1,55 @@
 import { GraphQLClient } from 'graphql-request'
```


would love to refactor this test to remove the use of `graphql-request`, but maybe that's not your priority at the moment... we could help with that, @aonomike is getting awesome at this... maybe in a separate `PR` though?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-10-14T13:26:07Z" title="Monday, October 14th 2019, 3:26:07 pm +02:00">Oct 14, 2019</time>_

---


You are right @mattwr18 . I set the priority to get this PR done. Would love if @aonomike jumps into that … 😍